File: examples/python/display_from_weighted_edge_list.py
from webweb import Web

# edge weights are the third element in the edge
adjacency_list = [[0, 1, .1], [1, 2, .5], [2, 3, 1], [3, 4, 2]]

# create the web
web = Web(adjacency_list)

# we'll scale edge widths by weight to create a visual difference
web.display.scaleLinkWidth = True

# open the browser with the result
web.draw()

File: examples/matlab/display_from_weighted_edge_list.m
% make a matrix of weighted edges
% edge weights are the third element in the edge
edge_list = [...
    1, 2, .1;... 
    2, 3, .5;... 
    3, 4, 1;... 
    4, 5, 2;...
];

% call webweb
webweb(edge_list);

%% [OPTIONAL] Set default: show weighted edges.
% To set a default parameter, we need to creat a webweb struct
% and place our parameter setting in it.
% Place this edgeList into a network within the webweb struct
ww.networks.network.edgeList = edge_list;
% Set scaleLinkWidth to True. 
ww.display.scaleLinkWidth = 'True';
% call webweb
webweb(ww);
